### PR TITLE
Feature/add pre trigger recipe

### PR DESCRIPTION
**Pull request by eramirez392**

### CODE REVIEW

File: recipes-multimedia/gstreamer/gst-pre-trigger_1.0.bb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+SUMMARY = "PreTriger"
```
Comment (rrcarlosrodriguez) [inline recipes-multimedia/gstreamer/gst-pre-trigger_1.0.bb]: There's a typo in there, please change to Pre-Record, which would be the official name

File: README.md [hunk 2]
```diff
@@ -18,14 +18,16 @@ cd $YOCTO_DIRECTORY/build/conf/
...
 
 - Go to your yocto directory and export the machine used. For example:
 ```
 export MACHINE=zcu106-zynqmp
 ```
 
-- Finally build both recipes
+## Building Gst-Shark and Gstreamer Deamon
+
+- Finally recipes
```
Comment (rrcarlosrodriguez) [inline README.md]: It's not clear, the comment could be removed

File: README.md [hunk 3]
```diff
@@ -46,4 +48,22 @@ bitbake gstd gst-shark
 bitbake gst-webrtc
 ```
 - For more information: 
-    http://developer.ridgerun.com/wiki/index.php?title=GstWebRTC_-_Building_GstWebRTC
\ No newline at end of file
+    http://developer.ridgerun.com/wiki/index.php?title=GstWebRTC_-_Building_GstWebRTC#Building_Gst-WebRTC_on_Yocto
+
+## Building Pre-Trigger (Private)
```
Comment (rrcarlosrodriguez) [inline README.md]: Pre-Record

File: README.md [hunk 4]
```diff
@@ -46,4 +48,22 @@ bitbake gstd gst-shark
...
+
+- Pre-Trigger is an add-on to RidgeRun's professional SDK. You can purchase Pre-Trigger with full source code, from:
```
Comment (rrcarlosrodriguez) [inline README.md]: Pre-Record

File: README.md (hunk 2 above)
Comment (eramirez392) [reply README.md]: I'm sorry, there's a typo

**Review by rrcarlosrodriguez — CHANGES_REQUESTED** [review]
The plugin name should be pre-record, please change.

File: README.md [hunk 5]
```diff
@@ -46,4 +48,22 @@ bitbake gstd gst-shark
 bitbake gst-webrtc
 ```
 - For more information: 
-    http://developer.ridgerun.com/wiki/index.php?title=GstWebRTC_-_Building_GstWebRTC
\ No newline at end of file
+    http://developer.ridgerun.com/wiki/index.php?title=GstWebRTC_-_Building_GstWebRTC#Building_Gst-WebRTC_on_Yocto
```
Comment (michaelgruner) [inline README.md:51]: Wrong wiki

File: recipes-multimedia/gstreamer/gst-pre-trigger_1.0.bb [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+DEPENDS = "gstreamer1.0 gstreamer1.0-plugins-base libffi"
+
+SRCBRANCH ?= "master"
+SRCREV = "64ae60cc8bc10bc75bc464da71d325a7147d089e"
+SRC_URI = " git://git@gitlab.com/RidgeRun/orders/<Customer-Directory>/gst-pre-trigger.git;protocol=ssh;branch=${SRCBRANCH}"
+
+S = "${WORKDIR}/git"
+
+FILES_${PN} += "\
+        ${libdir}/gstreamer-1.0/libgstpretrigger.so \
```
Comment (michaelgruner) [inline recipes-multimedia/gstreamer/gst-pre-trigger_1.0.bb]: These should be renamed to pre-record.

**Review by michaelgruner — CHANGES_REQUESTED** [review]

**Pull request merged**